addons/survey/static/src/scss/survey_survey_views.scss

```diff
@@ -174,3 +174,8 @@
         }
     }
 }
+
+.o_form_label.o_form_label_readonly.o_survey_label_survey_start_url{
+    opacity: 100;
```

Comment:
Opacity takes values from 0 to 1, so 100 is a bit much ^^

Anyway you could try using the "opacity-100" bootstrap class? It seems available. 
That way we avoid the custom class (and font-weight does not seem to change anything).